src/backend/Interner.zig

```diff
@@ -71,6 +73,12 @@ pub const Key = union(enum) {
         f80: f80,
         f128: f128,
     };
+    pub const Complex = union(enum) {
+        cf32: [2]Ref,
+        cf64: [2]Ref,
+        cf80: [2]Ref,
+        cf128: [2]Ref,
```

Comment:
Since these are complex floats of specific size you should be able to use the specific float types.
```suggestion
        cf32: [2]f32,
        cf64: [2]f64,
        cf80: [2]f80,
        cf128: [2]f128,
```
Integer support can then be added later in a similar manner.